src/Faker/Provider/Imei.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Faker\Provider;
+
+use Faker\Calculator\Luhn;
+
+class Imei extends Base
```

Comment:
@fzaninotto. the IMEI is used in some mobile devices, not just Phone. Do you think that move to PhoneNumber Provider is necessary?